src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -292,6 +301,26 @@ internal static SecurityKey GetSecurityKey(EncryptingCredentials encryptingCrede
 
                 securityKey = encryptingCredentials.Key;
             }
+#if NET472
+            else if (SupportedAlgorithms.EcdsaWrapAlgorithms.Contains(encryptingCredentials.Alg))
+            {
+                // on decryption we get the public key from the EPK value see: https://datatracker.ietf.org/doc/html/rfc7518#appendix-C
+                string apu = null, apv = null;
+                if (additionalHeaderClaims != null && additionalHeaderClaims.Count > 0)
+                {
+                    if (additionalHeaderClaims.TryGetValue(JwtHeaderParameterNames.Apu, out object objApu))
+                        apu = objApu.ToString();
```

Comment:
add a new line after 312 to separate clauses.

Reply:
Added